src/api/auth/getUserData/getUserData.ts

```diff
@@ -0,0 +1,15 @@
+import firestore from '@react-native-firebase/firestore';
+import {TUser} from '../../../xstate/User/types';
```

Comment:
btw polecałbym machnąć aliasy na ścieżki, bo teraz to jak gówno wygląda

Reply:
właśnie będę musiał ogarnąć te importy ogólnie bo po 1 jak importuję coś swojego np to mi daje "@/xstate/User/types" i potem wali mi błędami a po 2 jak chcę coś importować z bibliotek to średnio podpowiada

Reply:
To zesrałeś coś

Reply:
done